test/e2e/elasticsearch_test.go

```diff
@@ -179,6 +184,17 @@ func getJaegerSimpleProdWithServerUrls() *v1.Jaeger {
 			},
 		},
 	}
+
+	if useOtelCollector {
+		logrus.Infof("Using OTEL collector for %s", name)
+		otelOptions := getOtelCollectorOptions()
+		otelCollectorSpec := &v1.JaegerCollectorSpec{
+			Image:  otelCollectorImage,
+			Config: v1.NewFreeForm(otelOptions),
+		}
+		exampleJaeger.Spec.Collector = *otelCollectorSpec
```

Comment:
Some tests just set the image and config and others set the whole spec? Can we unify them to use the same approach?